src/redux/Books/books.js

```diff
@@ -1,36 +1,84 @@
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/OyUiXqyBVJcIT8DxplTW/books/';
+
 const ADD = 'redux/Books/book/ADD';
 const REMOVE = 'redux/Books/book/REMOVE';
+const GET = 'redux/Books/book/GET';
 const initialState = [];
-
 export default function bookReducer(state = initialState, action) {
   switch (action.type) {
     case 'redux/Books/book/ADD':
       return [
         ...state,
-        {
-          id: action.id,
-          title: action.payload.title,
-          author: action.payload.author,
-        },
+        action.payload,
       ];
 
     case 'redux/Books/book/REMOVE':
-      return state.filter((book) => book.id !== action.id);
+      return state.filter((book) => book.id !== action.payload);
+
+    case 'redux/Books/book/GET':
+      return action.payload;
 
     default: return state;
   }
 }
 
-const addBook = (id, payload) => ({
-  type: ADD,
-  payload,
-  id,
-});
+const addBook = (book) => (dispatch) => {
+  fetch(URL, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: book.id,
+      title: book.title,
+      author: book.author,
+      category: 'Fiction',
+    }),
+  }).then(() => {
+    dispatch({
+      type: ADD,
+      payload: {
+        id: book.id, title: book.title, author: book.author,
+      },
+    });
+  });
+};
 
-const remBook = (id) => ({
-  type: REMOVE,
-  id,
+const remBook = (id) => (dispatch) => {
+  const URlId = `${URL}${id}`;
+  fetch(URlId, {
+    method: 'DELETE',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+    }),
+  }).then(() => {
+    dispatch({
+      type: REMOVE,
+      payload: id,
+    });
+  }).catch((erorr) => console.log(erorr));
```

Comment:
* Kindly remove all console.log statement from your codebase as it isn't best practice also sensitive informations might be logged into the console